device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -65,7 +96,9 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
 
     // If there are no cells, return right away.
     if (num_cells == 0) {
-        return measurements;
+        return {std::move(measurements),
+                traccc::edm::silicon_cluster_collection::buffer{
+                    std::vector<unsigned int>{}, m_mr.main}};
```

Comment:
`vecmem::edm::buffer` has a default constructor. You can use that here.

Reply:
:+1: